src/common/local_file_system.cpp

```diff
@@ -919,7 +919,18 @@ vector<string> LocalFileSystem::Glob(const string &path, FileOpener *opener) {
 	if (absolute_path) {
 		// for absolute paths, we don't start by scanning the current directory
 		previous_directories.push_back(splits[0]);
+	} else {
+		// If file_search_path is set, use those paths as the first glob elements
+		Value value;
+		if (opener->TryGetCurrentSetting("file_search_path", value)) {
```

Comment:
The `opener` is not guaranteed to be there in all cases, particularly this seems to cause a problem in a specific unit test:

```cpp
/home/runner/work/duckdb/duckdb/src/common/local_file_system.cpp:925:35: runtime error: member call on null pointer of type 'struct FileOpener'
[25](https://github.com/duckdb/duckdb/actions/runs/3275083989/jobs/5390740031#step:6:26)
```

Perhaps add a check that the `opener` is defined as well? (i.e. `opener && opener->...`)